rtgui/preferences.cc

```diff
@@ -509,14 +509,14 @@ Gtk::Widget* Preferences::getProcParamsPanel ()
     Gtk::HBox* hb42 = Gtk::manage (new Gtk::HBox ());
     darkFrameDir = Gtk::manage (new Gtk::FileChooserButton (M ("PREFERENCES_DIRDARKFRAMES"), Gtk::FILE_CHOOSER_ACTION_SELECT_FOLDER));
     Gtk::Label *dfLab = Gtk::manage (new Gtk::Label (M ("PREFERENCES_DIRDARKFRAMES") + ":"));
-    hb42->pack_start (*dfLab , Gtk::PACK_SHRINK, 4 );
+    hb42->pack_start (*dfLab, Gtk::PACK_SHRINK, 4 );
     hb42->pack_start (*darkFrameDir, Gtk::PACK_EXPAND_WIDGET, 4);
     dfLabel = Gtk::manage (new Gtk::Label ("Found:"));
     Gtk::VBox* vbdf = Gtk::manage (new Gtk::VBox ());
     vbdf->pack_start ( *hb42, Gtk::PACK_SHRINK, 4);
     vbdf->pack_start ( *dfLabel, Gtk::PACK_SHRINK, 4 );
     fdf->add ( *vbdf );
-    mvbpp->pack_start ( *fdf , Gtk::PACK_SHRINK, 4);
+    mvbpp->pack_start ( *fdf, Gtk::PACK_SHRINK, 4);
```

Comment:
@Beep6581 
> do you think it would be a good idea to run all of the code files through astyle 3.0.1 to get them consistent? Then I could manually fix what needs to be fixed, like the wbEntries in procparams.cc.

No. I don't know of any code formatter who does it all right. You'd have to review and fix all the files. That time could be spent better. 😉 

I tried `astyle` in the past on my [cache](https://github.com/Beep6581/RawTherapee/blob/dev/rtengine/cache.h) implementation and code with lambdas, and it made some uncomprehesible changes. Maybe v3 is better now in that respect, but I doubt it.

> Second question, would you be willing to make the changes to rawtherapee.astylerc based on the docs to improve the formatting?

Sure. The thing is, we haven't defined what is "right", so I can only tackle things that are obviously wrong (like the space after the brace above). I also think that an empty line after an opening curly brace is wrong, but we have many places implemented like this. So how do I know this is wrong?